test/authentication/authentication.test.js

```diff
@@ -64,6 +64,17 @@ describe('auth0.authentication', function() {
       });
     });
 
+    ['username', 'popupOptions', 'domain', 'tenant', 'timeout'].forEach(function(param) {
+      it.only('should remove parameter: ' + param, function() {
```

Comment:
isn't this `only` making the test script only run this case?

Reply:
FUUUUUUU

Reply:
<3